api-module-library/ironclad/test/api.test.js

```diff
@@ -195,15 +125,67 @@ describe('Ironclad API class', () => {
             expect(response).to.have.property('roles');
             expect(response).to.have.property('approvals');
             expect(response).to.have.property('signatures');
-            expect(response).to.have.property('isRevertibleToReview')
-        })
+            expect(response).to.have.property('isRevertibleToReview');
+        });
 
         it('should list all workflow approvals', async () => {
             const response = await api.listAllWorkflowApprovals(workflowID);
             expect(response).to.have.property('workflowId');
             expect(response).to.have.property('title');
             expect(response).to.have.property('approvalGroups');
             expect(response).to.have.property('roles');
-        })
-    })
+        });
+    });
+
+    describe('Records', () => {
+        let recordID;
+
+        it('should create a record', async () => {
+            const body = {
+                type: 'nDAs',
+                name: 'Example Record',
+                properties: {
+                    agreementDate: {
+                        type: 'date',
+                        value: '2022-08-12T00:00:00-07:00',
+                    },
+                    counterpartyName: {
+                        type: 'string',
+                        value: 'John Doe',
+                    },
+                },
+            };
+            const response = await api.createRecord(body);
+            expect(response).to.have.property('id');
+            expect(response).to.have.property('type');
+            expect(response).to.have.property('name');
+            expect(response).to.have.property('lastUpdated');
+        });
+
+        it('should list all records', async () => {
+            const response = await api.listAllRecords();
+            expect(response).to.have.property('page');
+            expect(response).to.have.property('pageSize');
+            expect(response).to.have.property('count');
+            expect(response).to.have.property('list');
+            recordID = response.list[0].id;
+        });
+
+        it('should list all record schemas', async () => {
+            const response = await api.listAllRecordSchemas();
+            expect(response).to.have.property('properties');
+            expect(response).to.have.property('recordTypes');
+        });
+
+        it('should retrieve a record', async () => {
+            const response = await api.retrieveRecord(recordID);
+            expect(response).to.have.property('id');
+            expect(response).to.have.property('type');
+            expect(response).to.have.property('name');
+            expect(response).to.have.property('lastUpdated');
+            expect(response).to.have.property('properties');
+            expect(response).to.have.property('attachments');
+            expect(response).to.have.property('links');
```

Comment:
I suggest adding a test that deletes the created record. Since we're making live tests to the API, we probably want to clean up anything that we've added/modified while running our tests.